introduction/java_int_to_string/Solution.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Scanner;
+import java.security.*; //dont know what exactly is used, cant check properly
```

Comment:
This is not a good excuse to use wildcard.
I think the code uses the following:

    import java.security.Permission;
    import java.security.SecurityException;
    import java.security.SecurityManager;